protected/repositories/ProjectRepository.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+
+namespace prime\repositories;
+
+use prime\helpers\ModelHydrator;
+use prime\interfaces\AccessCheckInterface;
+use prime\models\ar\Permission;
+use prime\models\ar\Project;
+use prime\models\ar\read\Project as ProjectRead;
+use prime\models\forms\project\Create;
+use prime\models\forms\project\Update as ProjectUpdate;
+use prime\values\ProjectId;
+
+class ProjectRepository
```

Comment:
Somehow I feel like I would expect an interface

Reply:
It doesn't add much in this case. There won't be multiple implementations of a repository for one model.
Furthermore each repository will have different functions and those functions will have different arguments and return types.